tsconfig.json

```diff
@@ -12,6 +12,10 @@
     "skipLibCheck": true,
     "strict": true,
     "target": "ES2018",
-    "newLine": "lf"
-  },
+    "newLine": "lf",
+    "baseUrl": ".",
+    "paths": {
+      "clipanion/lib/platform": ["./sources/platform"]
```

Comment:
Is it fixed if we use `moduleResolution: "node16"`?

Reply:
That will bring many (import) changes. Should we still try?

Reply:
Since we don't build with TSC, it depends on Rollup. And the latest version (v3) break bundle :thinking:

Reply:
it looks fine just replace `clipanion/lib/platform` with `clipanion/platform` to eliminate inconsistency between import and output in the source. This will be a breaking change.

Reply:
how about removing `lib` prefix from all exported paths?